src/main/java/com/techsophy/tsf/account/service/impl/UserFormDataServiceImpl.java

```diff
@@ -47,13 +50,15 @@ public class UserFormDataServiceImpl implements UserFormDataService
     private final IdGeneratorImpl idGenerator;
     private final TokenUtils tokenUtils;
     private final UserDetails userDetails;
+    private final Logger logger = Logger.getLogger("Inside UserFormDataService");
```

Comment:
Logger logger= LoggerFactory.getLogger(this.getClass());
that is how it should be used. or use annotation slf4j